### PR TITLE
Mailing List Process

### PR DESCRIPTION
**Pull request by KeiranWDigital**

### CODE REVIEW

File: pages-admin/mailinglist-process.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+permalink: /mailinglist-process/
+layout: default
+suffixes:
+- Assistance,
+- Power,
+- Resources,
+- Responsibility,
+- Us
+title: Event Process
```
Comment (dangerworm) [inline pages-admin/mailinglist-process.html]: Needs to be Mailing List Process

File: pages-admin/mailinglist-process.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+sitemap: false
+---
+
+<h1>Mailing List Process</h1>
+
+<p>Anyone can submit a message to be sent to the mailing list, to be sent either in a specific email or in the next newsletter</p>
```
Comment (dangerworm) [inline pages-admin/mailinglist-process.html]: Full stop at the end.

File: pages-admin/mailinglist-process.html [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+<p>Submissions should be sent to any committee member</p>
```
Comment (dangerworm) [inline pages-admin/mailinglist-process.html]: Full stop at the end.

File: pages-admin/mailinglist-process.html [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+<h2>Use of the mailing list</h2>
+
+<p>If event details are in the submitted draft, it must have followed the <a href="{{site.url}}/event-process"></a>CodeWith Event Process.</a></p>
```
Comment (dangerworm) [inline pages-admin/mailinglist-process.html]: Move the full stop outside the anchor please.

**Review by dangerworm — CHANGES_REQUESTED** [review]

**Comment by KeiranWDigital** [comment]
Changes made

**Review by dangerworm — APPROVED** [review]

**Pull request merged**